### ISSUE TITLE
Probability maps of some regions not loading

### ISSUE THREAD

**Issue by dickscheid** [FZJ-INM1-BDA/siibra-explorer#1002]
When selecting some areas - for example, "LB (Amygdala) right hemisphere", no probability map is shown in the viewer, despite one being available in siibra-python:

```python
import siibra
from nilearn import plotting
atlas = siibra.atlases['human']
lb_r = atlas.get_region('lb right')
pmap = lb_r.get_regional_map('mni152',maptype='continuous')
plotting.plot_stat_map(pmap.fetch())
```

After selecting this area in siibra-explorer, it keeps just showing the label mask:
![grafik](https://user-images.githubusercontent.com/1436846/131848724-93a3a0c5-d800-4be1-b1ec-2a4061b4e0a0.png)

**Comment by dickscheid** [comment]
output of the siibra-python snippet above:

![grafik](https://user-images.githubusercontent.com/1436846/131848858-3428e8af-de16-472c-91a9-033d7959dba6.png)

**Comment by xgui3783** [comment]
issue seems to be fixed (likely related to ebrains tokens, so also resolved by https://github.com/FZJ-INM1-BDA/siibra-api/releases/tag/0.0.2) 

![Screenshot from 2021-09-29 09-36-43](https://user-images.githubusercontent.com/19381783/135223898-354ace7d-e225-49c8-b373-fd4292eacac5.png)

will leave issue open until a monitoring step is setup

**Comment by xgui3783** [comment]
fixed as of 2.7.6
